src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores-export-button.component.ts

```diff
@@ -110,6 +170,69 @@ export class ExerciseScoresExportButtonComponent {
             }) || []
         );
     }
+
+    /**
+     * Retrieves a list of test cases names contained in a result's feedback list
+     * @param results list of results to extract the test case names from
+     * @private
+     */
+    private getTestCaseNamesFromResults(results: ResultWithPointsPerGradingCriterion[]): string[] {
```

Comment:
Could we move this into a the result-service or an util file, since this would probably be helpful in other places as well

Reply:
Both functions have now been moved to `result.utils.ts` 🙂